vb_toolbox/app.py

```diff
@@ -141,7 +148,7 @@ def main():
         cort_index = np.array(labels, np.bool)
         Z = np.array(cort_index, dtype=np.int)
         try:
-            result = vb.vb_cluster(vertices, faces, n_cpus, data, Z, args.norm[0], args.output[0] + "." + args.norm[0], nib_surf)
+            result = vb.vb_cluster(vertices, faces, n_cpus, data, Z, args.norm[0], args.tol[0], args.maxiter[0], args.output[0] + "." + args.norm[0], nib_surf)
```

Comment:
Following my previous comments here should be: args.tol and args.maxiter (no need to use a list). I think the same applies to norm